modules/prioritizing-and-managing-scanned-cves-across-images-and-deployments.adoc

```diff
@@ -0,0 +1,149 @@
+
+:_mod-docs-content-type: PROCEDURE
+[id="prioritizing-and-managing-scanned-cves-across-images-and-deployments_{context}"]
+= Prioritizing and managing scanned CVEs across images and deployments
+
+[role="_abstract"]
+
+By viewing the *Workload CVEs* page, you can get information about the vulnerabilities in applications running on clusters in your system. You can view vulnerability information across images and deployments. 
+
+The *Workload CVEs* page provides more advanced filtering capabilities than the dashboard, including the ability to view images and deployments with vulnerabilities and filter based on image, deployment, namespace, cluster, CVE, component, and component source.
+
+.Procedure
+
+. In the {product-title-short} portal, click *Vulnerability Management* -> *Workload CVEs*.
+
+. Choose the appropriate method to navigate through the images and deployments from the drop-down list, which is in the upper left of the page:
+** To view the images and deployments with observed CVEs, select *Image vulnerabilities*.
+** To view the images and deployments without observed CVEs, select *Images without vulnerabilities*.
+
+. Optional: Choose the appropriate method to re-organize the information in the *Workload CVEs* page:
+** To sort the table in ascending or descending order, select a column heading.
+** To filter the table, use the filter bar.
+** To select the categories that you want to display in the table, perform the following steps:
+... Click *Manage columns*.
+... Choose the appropriate method to manage the columns:
+**** To view all the categories, click *Select all*.
+**** To reset to the default categories, click *Reset to default*.
+**** To view only the selected categories, select the one or more categories that you want to view.
+
+** To filter CVEs based on an entity, select the appropriate filters and attributes.
++
+To select multiple entities and attributes, click the right arrow icon to add another criteria. Depending on your choices, enter the appropriate information such as text, or select a date or object.
++
+The filter entities and attributes are listed in the following table.
++
+.CVE filtering
+[cols="2",options="header"]
+|===
+|Entity|Attributes
+
+|Image
+a|
+* *Name*: The name of the image.
+* *Operating system*: The operating system of the image.
+* *Tag*: The tag for the image.
+* *Label*: The label for the image.
+* *Registry*: The registry where the image is located.
+|CVE
+a|
+* *Name*: The name of the CVE.
+* *Discovered time*: The date when {product-title-short} discovered the CVE.
+* *CVSS*: The severity level for the CVE.
++
+The following values are associated with the severity level for the CVE:
++
+** *is greater than*
+** *is greater than or equal to*
+** *is equal to*
+** *is less than or equal to*
+** *is less than*
+* *NVD CVSS*: The National Vulnerability Database (NVD) provides a CVSS score between `0` to `10` for a CVE based on the severity of the vulnerability.
```

Comment:
we dont have filter for nvd cvss currently.

Reply:
@agantony  I would not not recommend to remove it since screenshot helps understand the workflow better. But I will let you and @kcarmichael08  make final decision